test/testsets.json

```diff
@@ -31,40 +29,42 @@
-    { "name": "test_fs2.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_gpio1.js", "skip": ["all"], "reason": "needs hardware" },
-    { "name": "test_gpio2.js", "skip": ["all"], "reason": "need user input"},
-    { "name": "test_http_get.js", "timeout": 20, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_http_header.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpclient_timeout.js", "timeout": 10, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpclient_timeout2.js", "timeout": 15, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpserver_timeout.js", "timeout": 10, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_httpserver.js", "timeout": 20, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_fs_event.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_fs_open_read_async.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_fs_open_read_sync.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_gpio_input.js", "skip": ["all"], "reason": "needs hardware" },
+    { "name": "test_gpio_output.js", "skip": ["all"], "reason": "need user input"},
     { "name": "test_i2c.js", "skip": ["all"], "reason": "need to setup test environment" },
     { "name": "test_module_cache.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
-    { "name": "test_net1.js" },
-    { "name": "test_net2.js" },
-    { "name": "test_net3.js", "timeout": 20, "skip": ["nuttx"], "reason": "requires too many socket descriptors and too large buffers" },
-    { "name": "test_net4.js" },
-    { "name": "test_net5.js" },
-    { "name": "test_net6.js" },
-    { "name": "test_net7.js", "skip": ["nuttx"], "reason": "requires too many socket descriptors" },
-    { "name": "test_net8.js" },
-    { "name": "test_net9.js" },
-    { "name": "test_net10.js" },
-    { "name": "test_next_tick.js" },
+    { "name": "test_net_1.js" },
+    { "name": "test_net_2.js" },
+    { "name": "test_net_3.js", "timeout": 20, "skip": ["nuttx"], "reason": "requires too many socket descriptors and too large buffers" },
+    { "name": "test_net_4.js" },
+    { "name": "test_net_5.js" },
+    { "name": "test_net_6.js" },
+    { "name": "test_net_7.js", "skip": ["nuttx"], "reason": "requires too many socket descriptors" },
+    { "name": "test_net_8.js" },
+    { "name": "test_net_9.js" },
+    { "name": "test_net_10.js" },
+    { "name": "test_net_headers.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_net_http_get.js", "timeout": 20, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
+    { "name": "test_net_httpclient_timeout.js", "timeout": 10, "skip": ["nuttx"], "reason": "not implemented for nuttx" },
```

Comment:
I suggest to rename this to `test_net_httpclient_timeout_1`. For features having more than one test case, it will be clearer to have a rule to put its order.